Patches/RimWorld - Witcher Monster Hunt/Projectile_WH.xml

```diff
@@ -8,12 +8,12 @@
 		<match Class="PatchOperationSequence">
 			<operations>
 
-			<li Class="PatchOperationReplace">
+			<li Class="PatchOperationAdd">
 				<xpath>/Defs/ThingDef[defName="WMH_LeshyWave" or
 				defName="WMH_WyvernAcidSpit" or
 				defName="WMH_Rock" or
```

Comment:
The Rock is getting a Duplicate for the ThingClass. Will need a replace instead of an add. Looks good otherwise, though.

Reply:
All done.